src/components/Layout/MovieItem/MovieItem.tsx

```diff
@@ -0,0 +1,24 @@
+import './MovieItem.scss';
+
+export interface MovieItemProp {
```

Comment:
I believe the interface name should end with `s` in this case, `MovieItemProps`

Reply:
ok